proposals/lip-0063.md

```diff
@@ -428,7 +429,10 @@ def createSupplySubstoreArray():
         lockedBalances = getLockedBalances(account)
         if lockedBalances is not empty:
             totalLSKSupply += lockedBalances[0].amount
-    LSKSupply = {"tokenID": TOKEN_ID_LSK, "totalSupply": totalLSKSupply}
+    legacyReserveAccount = account with account.address == ADDRESS_LEGACY_RESERVE
```

Comment:
This is a bit confusing as `accounts` and `legacyAccounts` contain the state data from Lisk Core v3 and only `tokenObj.userSubstore` contains the accounts and locked tokens in the new format. However, `tokenObj.userSubstore` is not an input parameter to the function at the moment.

Reply:
Indeed, the logic or legacy reserve account was wrong because of mixing up v3 accounts with v4 accounts. Now it is hopefully fixed.

Reply:
Lookds good now.